src/main/java/mezz/jei/transfer/RecipeTransferErrorTooltip.java

```diff
@@ -17,9 +17,9 @@
 public class RecipeTransferErrorTooltip implements IRecipeTransferError {
 	private final List<ITextComponent> message = new ArrayList<>();
 
-	public RecipeTransferErrorTooltip(String message) {
+	public RecipeTransferErrorTooltip(ITextComponent message) {
 		this.message.add(new TranslationTextComponent("jei.tooltip.transfer"));
-		StringTextComponent messageTextComponent = new StringTextComponent(message);
+		IFormattableTextComponent messageTextComponent = message.deepCopy();
```

Comment:
This does a deep copy here because there is a chance a text component that is not supposed to be formattable is passed here. If you would rather I can make the transfer error helpers take formattable text components and specify in the JavaDoc that they are supposed to be allowed to be formatted.

Reply:
deepCopy is fine, I think if there is something like this that is not compiler-enforceable coming from the API then we should just be careful with it like this.